modules/labs-react/text-input/lib/TextInput.Label.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+
+import {
+  accessibleHide,
+  createComponent,
+  styled,
+  StyledType,
+  Themeable,
+  useModelContext,
+} from '@workday/canvas-kit-react/common';
+import {type} from '@workday/canvas-kit-react/tokens';
+import {HStack} from '@workday/canvas-kit-labs-react/layout';
+
+import {TextInputModel} from './useTextInputModel';
+import {TextInputModelContext} from './TextInput';
+
+export interface TextInputLabelProps extends Themeable {
+  model?: TextInputModel;
+  /**
+   * If the field is required, provide the title required label.
+   */
+  isRequiredLabel?: string;
```

Comment:
Seemed redundant to have required and required label, but wondering if it would make more sense to make 'isRequired' part of the model state and make this field have a default value of "required"